src/Symfony/Component/PropertyInfo/Tests/Fixtures/NullExtractor.php

```diff
@@ -57,6 +58,14 @@ public function getType($class, $property, array $context = []): ?Type
         return null;
     }
 
+    public function getAttributes($class, $property, array $context = []): ?array
+    {
+        $this->assertIsString($class);
+        $this->assertIsString($property);
+
+        return null;
+    }
```

Comment:
```suggestion
    public function getAttributes(string $class, string $property, array $context = []): ?array
    {
        return null;
    }
```

I think that is enough, thanks to modern PHP typing system.